### PR TITLE
updated the necessary commands

### PR DESCRIPTION
**Pull request by mholder6**
## Description
updated the necessary commands related to the location of files

## Testing
work through the QuickStart to ensure the commands work

### CODE REVIEW

File: opendatahub/quickstart/basic/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ spec:
 
 Deploy the sample model:
 ~~~
-./deploy.sh
+./opendatahub/quickstart/basic/deploy.sh
```
Comment (Jooho) [inline opendatahub/quickstart/basic/README.md:54]: each folder has to have deploy.sh file (https://github.com/opendatahub-io/modelmesh-serving/tree/main/opendatahub/quickstart) 
and the one of pre-requisite docs explains how to use the script.
https://github.com/opendatahub-io/modelmesh-serving/blob/main/opendatahub/quickstart/common_docs/modelmesh-install.md

~~~
source ../env.sh

cd opendatahub/quickstart/$QUICK_START_FOLDER # (for example, cd opendatahub/quickstart/basic)

./deploy.sh
~~~
So I don't think this is needed.

File: opendatahub/quickstart/basic/README.md [hunk 2]
```diff
@@ -85,10 +85,19 @@ The following `curl` examples demonstrate how to perform inference requests.
...
+export HOST_URL=$(oc get route example-onnx-mnist -ojsonpath='{.spec.host}' -n modelmesh-serving)
+
+export HOST_PATH=$(oc get route example-onnx-mnist -ojsonpath='{.spec.path}' -n modelmesh-serving)
+
+##If you give commands echo $HOST_URL and echo $HOST_PATH you should get a similar output to the following:
+echo $HOST_URL
+example-onnx-mnist-modelmesh-serving.apps.user.dev.datahub.redhat.com
+echo $HOST_PATH
+/v2/models/example-onnx-mnist
+
```
Comment (Jooho) [inline opendatahub/quickstart/basic/README.md:97]: this one is the same. please check [this doc](https://github.com/opendatahub-io/modelmesh-serving/blob/main/opendatahub/quickstart/common_docs/modelmesh-install.md)
Especially this part
~~~
source ../env.sh

cd opendatahub/quickstart/$QUICK_START_FOLDER # (for example, cd opendatahub/quickstart/basic)

./deploy.sh
~~~

File: opendatahub/quickstart/basic/README.md [hunk 3]
```diff
@@ -85,10 +85,19 @@ The following `curl` examples demonstrate how to perform inference requests.
...
 
-curl --silent --location --fail --show-error --insecure https://${HOST_URL}${HOST_PATH}/infer -d  @${COMMON_MANIFESTS_DIR}/input-onnx.json
+cd opendatahub/quickstart/common_manifests/
+curl -k --insecure https://${HOST_URL}${HOST_PATH}/infer -d  @input-onnx.json
```
Comment (Jooho) [inline opendatahub/quickstart/basic/README.md:100]: This part is the same as the above comment.

**Comment by Jooho** [comment]
@mholder6 do you still need this ticket?

**Comment by mholder6** [comment]
Hi Jooho,

I am not completely sure right now, I will ask Selbi and Vedant how they
want me to proceed on Monday.

**Comment by spolti** [comment]
closing since there is no interaction in this PR, if you still need to have this contribution added, feel free to reopen it.

Thanks.

**Pull request closed without merging**